process_css_extend_template.xml

```diff
@@ -4,38 +4,34 @@
 --><project xmlns:dita="http://dita-ot.sourceforge.net" name="fox.jason.extend.css">
     
     <target name="extend.css.init">
+        <condition property="user.csspath" value="" else="${args.csspath}/">
+            <or>
+                <not>
+                    <isset property="args.csspath"/>
+                </not>
+                <isabsolute path="${args.csspath}"/>
+            </or>
+        </condition>
         <mkdir dir="${dita.temp.dir}"/>
-        <tempfile deleteonexit="true" destdir="${dita.temp.dir}" property="extend.css.file" suffix=".css"/>
-        <touch file="${extend.css.file}"/>
+        <tempfile deleteonexit="true" destdir="${dita.temp.dir}" property="extend.css.file" suffix=".css" createfile="yes"/>
```

Comment:
`<tempfile>` can `@createfile` so there's no need to `<touch>` separately

Reply:
Added